models/campaign.go

```diff
@@ -608,5 +629,24 @@ func CompleteCampaign(id int64, uid int64) error {
 	if err != nil {
 		log.Error(err)
 	}
+
+  //INFO send webhook
+  pl := map[string]interface{} {
```

Comment:
I’d hold off on sending campaign completion events for now. The proper solution would be to run “AddEvent” when a campaign is completed, though I’d see that out of scope for this PR- it’s something I can fix shortly after this is merged, then our webhook via AddEvent will fire.

Reply:
removed it